.circleci/config.yml

```diff
@@ -0,0 +1,53 @@
+version: 2.1
+
+workflows:
+  version: 2
+  commit:
+    jobs:
+      - build
+
+jobs:
+  build:
+    docker:
+      - image: circleci/node:12.16.3
+    resource_class: small
+    environment:
+      TOTAL_CPUS: 1 # For resource_class small
```

Comment:
can't you set the resource_class explicitly somewhere?

Reply:
This just sets an environment variable which is passed later to jest and flow. We had problems with on of those because it assumed the cpu count was different, not sure though which one made problems.